### DISCUSSION TITLE
[Ubuntu] How to resolve the error `No torrent specified!  Please use the -t option first.`

### DISCUSSION THREAD

**Discussion by shankhs** [transmission/transmission#2688] [Q&A] [answered]
Hi all,

I am invoking transmission-daemon as:
```
transmission-remote --auth USER:PASSWORD -w /home/USER/ -g ~/.config/transmission-daemon/settings.json -l
```
and I keep getting this error:
`No torrent specified!  Please use the -t option first.`. The complete output is as follows:
```
No torrent specified!  Please use the -t option first.
localhost:9091/transmission/rpc/ responded: "success"
ID     Done       Have  ETA           Up    Down  Ratio  Status       Name
Sum:              None               0.0     0.0
localhost:9091/transmission/rpc/ responded: "success"
```
But when I invoke it with 
```
transmission-remote --auth USER:PASSWORD -w /home/USER/ -l
```
I dont get the error.

Any idea what is going on or any tips on how to debug this?

Thanks!

**Comment by fredo-47** [comment, marked as answer]
For transmission-remote, the -g option is different from transmission-daemon.

Using t-remote, you don‘t need to specify the path to settings.json, because the running daemon already knows it from when itself got started.

For t-remote, -g defines which files to (mark for) download, and it naturally needs to know which torrent(s) it should refer to ;-)
